README.md

```diff
@@ -77,6 +77,24 @@ bin/elixirc lib/elixir/lib/string.ex -o lib/elixir/ebin
 bin/elixir lib/elixir/test/elixir/string_test.exs
 ```
 
+To recompile only changed files (including Erlang modules):
```

Comment:
Are you sure? When I run `make compile` on my machine it only outputs recompilation for changed files.

`➜  elixir git:(master) make clean`

```
➜  elixir git:(master) make compile
==> elixir (compile) # takes a couple of minutes
Compiled src/elixir_parser.yrl
Compiled src/elixir.erl
... lots more ...
```

And again:

```
➜  elixir git:(master) make compile
==> elixir (compile) # takes a couple of seconds
➜  elixir git:(master)
```